addons/account/static/src/helpers/account_tax.js

```diff
@@ -175,13 +175,13 @@ export function prepare_taxes_computation(tax_values_list, kwargs={}) {
 
     // Flatten the taxes and order them.
     const sorted_tax_values_list = tax_values_list.sort(
-        (v1, v2) => v1.sequence - v2.sequence || v1.tax_id - v2.tax_id
+        (v1, v2) => v1.sequence - v2.sequence || v1.id - v2.id
     );
     let flatten_tax_values_list = [];
     for (const tax_values of sorted_tax_values_list) {
         if (tax_values.amount_type === "group") {
             const sorted_children_tax_ids = tax_values._children_tax_ids.sort(
-                (v1, v2) => v1.sequence - v2.sequence || v1.tax_id - v2.tax_id
+                (v1, v2) => v1.sequence - v2.sequence || v1.id - v2.id
```

Comment:
How come it was working before? :thinking: 
Nothing changed apart from the test framework, so the business code was bugged  but the tests passed?
The commit description is not very explicit :sweat_smile:

Reply:
@william-andre The test is checking the order of taxes based on sequences but not group of taxes having children having the same sequence